languages/minecraft/acf-minecraft_ru.properties

```
@@ -21,18 +21,18 @@
 #  WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 
-acf-minecraft.invalid_world = Ошибка: Указан несуществующий мир.
+acf-minecraft.invalid_world = Ошибка: Этот мир не существует.
 acf-minecraft.you_must_be_holding_item = Ошибка: Вы должны держать предмет в ведущей руке.
 acf-minecraft.player_is_vanished_confirm = \
     Предупреждение: <c2>{vanished}</c2> невидим. Не выдавайте его!\n\
-    Для подтверждения вашего действия, добавьте <c2>:confirm</c2> после его имени.\n\
+    Для подтверждения вашего действия, добавьте <c2>:confirm</c2> в конец его имени.\n\
     Пример: <c2>{vanished}:confirm</c2>
-acf-minecraft.username_too_short = Ошибка: Ник слишком короткий. Минимальная длина - три символа.
+acf-minecraft.username_too_short = Ошибка: Ник слишком короткий, должен быть не менее трех символов.
```

Может лучше
```suggestion
acf-minecraft.username_too_short = Ошибка: Ник должен быть не короче трех символов.
```

---

Меня уж сильно напрягает "не короче". В целом вариант неплохой, если бы не это...

---

Может перенести частицу "не" ближе к началу? Тогда получится:
```suggestion
acf-minecraft.username_too_short = Ошибка: Ник не должен быть короче трех символов.
```